tests/integration/sms/dialer_interaction_test.js

```diff
@@ -0,0 +1,49 @@
+/* global require, marionette, setup, suite, test, __dirname */
+'use strict';
+
+
+var assert = require('chai').assert;
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var requireFromApp = require(appRoot +
+  '/shared/test/integration/require_from_app');
+var Messages = requireFromApp('sms', 'lib/messages');
+var InboxView = requireFromApp('sms', 'lib/views/inbox/view');
+var ThreadGenerator = requireFromApp('sms', 'generators/thread');
+
+marionette('Dialer', function() {
+  var messagesApp;
+
+  var client = marionette.client({
+    prefs: {
+      'focusmanager.testmode': true
+    }
+  });
+
+  var thread;
+
+  setup(function() {
+    messagesApp = Messages.create(client);
+    client.contentScript.inject(
```

Comment:
nit: could you please move client.contentScript.inject to the end of the setup like we do in the rest of the tests and separate it with a new line from the rest of the code?